docs/content/latest/manage/backup-restore/point-in-time-restore.md

```diff
@@ -0,0 +1,137 @@
+
+Incremental backups should cover the following scenarios:
+
+* All changes as a result of DML statements such as INSERT, UPDATE, DELETE
+* DDL statements, such as creation of new tables and dropping of existing tables
+* Any updates for tables that may get dropped in that time interval
+
+This feature helps dealing with developer and operator error recovery (mentioned in the Scenarios section A).
+The restore should also include any DDL changes, such as create/drop/alter tables.
+The time granularity of the point in time that one can restore to (1 second, 1 minute etc) is a separate parameter / specification.
+Differential incremental backups require applying multiple incremental backups on top of a base backup
+
+Compared to flashbacks, incremental backups:
+
+* Often run more frequently, since the data set size is reduced.
+* Can handle a disaster-recovery scenario.
+
+There are two types of incremental backups, _differential_ and _cumulative_. Although YugayteDB supports both types, we recommend differential incremental backups.
+
+#### Differential incremental backups
+
+Each differential incremental backup only contains the updates that occurred after the previous incremental backup. All changes since last incremental. A point-in-time restore operation in this case would involve restoring the latest base backup, followed by applying every differential incremental backup taken since that base backup.
+
+#### Cumulative incremental backups
+
+Each cumulative incremental backup contains all changes since the last base backup. The timestamp of the last base backup is specified by the operator. In this case, the point-in-time restore operation involves restoring the latest base backup, followed by applying the latest cumulative incremental backup.
+
+## Use cases
+
+The following table provides a quick comparison of the intended usage patterns.
+
+| Scenario | In-cluster flashback DB | Off-cluster flashback DB | Incremental backup |
+| :------- | :---------------------- | :----------------------- | :----------------- |
+| **Disk/file corruption** | Handled by replication in cluster | Handled by replication in cluster | Handled by replication in cluster |
+| **App/operator error** | Yes | Yes | Yes |
+| **RPO** | Very low | High | Medium |
+| **RTO** | Very low | High | High |
+| **Disaster Recovery** | No (replication in cluster) | Yes | Yes |
+| **Impact / Cost** | Very low | High (snapshot and copy) | Medium |
+
+## Limitations
+
+This feature is in active development. YSQL and YCQL support different features, as detailed in the sections that follow.
+
+Development for this feature is tracked in [issue 7120](https://github.com/yugabyte/yugabyte-db/issues/7120). Some forthcoming features include:
+
+* Support for undoing YSQL metadata operations, such as CREATE, ALTER, TRUNCATE, or DROP TABLE
+* Options to restore with different granularities, such as a single YSQL database or the whole YCQL dataset.
```

Comment:
This is actually already in, so we can remove this.

I'd add maybe a note that automatic tablet splitting and PITR do not work well together.

Reply:
Removed, and added the note. Any specifics you want to add?